### ISSUE TITLE
Keyboard shortcut to focus Filter in Database Navigator

### ISSUE THREAD

**Issue by bhuynhdev** [dbeaver/dbeaver#40660]
**Is your feature request related to a problem? Please describe.**
When working in a database server with multiple databases (I use MySQL specifically), it is often needed to switch between databases. Therefore, a keyboard shortcut directly to the Search/Filter of the Database Navigator would greatly boost the productivity

This feature was once brought up in https://github.com/dbeaver/dbeaver/issues/15536, but was closed without a final contributor's response, so I would like to re-raise it.

**Describe the solution you'd like**
A keyboard shortcut to focus the following field. This keyboard shortcut can then be customized per user preferences.

<img width="297" height="90" alt="Image" src="https://github.com/user-attachments/assets/d69a57fa-777c-4c98-bf66-55b46428ebf5" />

**Describe alternatives you've considered**
No alternatives found at the moment

**Comment by ShadelessFox** [comment]
Thanks for the suggestion

**Comment by MukulGhare** [comment]
Hi 
I would like to work on this
could you please assign this to me ?

**Comment by ShadelessFox** [comment]
Hello @MukulGhare,

Sorry, another user just submitted a PR.
